openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -275,13 +275,13 @@ class Config:
     or `independent` (independently sampled lambda windows).
     Default `repex`.
     """
-    online_analysis_interval: Optional[int] = 200
+    online_analysis_interval: Optional[int] = 250
     """
     MCMC steps (i.e. ``IntegratorSettings.n_steps``) interval at which
     to perform an analysis of the free energies.
 
     At each interval, real time analysis data (e.g. current free energy
-    estimate and timing data) will be written to a yaml file named 
+    estimate and timing data) will be written to a yaml file named
```

Comment:
Can you update this to make it clear of the new limitation? i.e. must be a multiple of X

Reply:
Yes!